### PR TITLE
Preserve the coordinate reference system when returning geodataframes

### PR DESCRIPTION
**Pull request by apulverizer**
This sets the coordinate reference system to be the same as the source coordinate reference system of the trajectories.

### CODE REVIEW

**Comment by anitagraser** [comment]
Thank you for this contribution, Aaron! Could you please also include a test that checks that this is working as expected?

**Comment by apulverizer** [comment]
@anitagraser I updated the branch with new tests for this

File: movingpandas/tests/test_trajectory_aggregator.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    def test_get_significant_points_gdf_crs(self):
+        assert self.trajectory_aggregator.get_significant_points_gdf().crs == CRS_METRIC
+        assert self.trajectory_aggregator_latlon.get_significant_points_gdf().crs == CRS_LATLON
+
+    def test_get_flows_gdf_crs(self):
+        assert self.trajectory_aggregator.get_flows_gdf().crs == CRS_METRIC
+        assert self.trajectory_aggregator_latlon.get_flows_gdf().crs == CRS_LATLON
+
+    def test_get_clusters_gdf_crs(self):
+        assert self.trajectory_aggregator.get_flows_gdf().crs == CRS_METRIC
```
Comment (anitagraser) [inline movingpandas/tests/test_trajectory_aggregator.py:40]: The implementation of test_get_clusters_gdf_crs seems to be identical to test_get_flows_gdf_crs

Comment (apulverizer) [reply movingpandas/tests/test_trajectory_aggregator.py]: Fixed, sorry about that.

**Pull request merged**